requirements.txt

```diff
@@ -1,10 +1,8 @@
 -f https://cdn.research.pdx.edu/pypi/dist/
 django<1.8
 pymysql
--e git://github.com/PSU-OIT-ARC/djangocas.git#egg=djangocas
 pytz
-python3-ldap<0.9.6 # the higher versions don't install correctly
 -e git://github.com/PSU-OIT-ARC/django-perms.git#egg=django-perms
-django-arcutils>=1.2.0,<2.0
+django-arcutils[ldap]>=1.2.0,<2.0
```

Comment:
One thing to note here is that in eaf882b we only upgrade django-arcutils to `1.x-maint` without including ldap, whereas here we replace `python3-ldap` by installing `ldap3` with ARCutils, given by the `[ldap]` tag.